packages/allure-playwright/src/index.ts

```diff
@@ -203,6 +218,14 @@ class AllureReporter implements Reporter {
         runtime.writeAttachment(stderr, "text/plain"),
       );
     }
+
+    allureTest.historyId = md5(
```

Comment:
is not required, it will be calculated this way on the report/testops site

Reply:
Is it ok i if i completely remove historyId from playwright integration? previously we used it [there](https://github.com/allure-framework/allure-js/blob/a1a1132128a346ac8453e206b287f63931e999bc/packages/allure-playwright/src/index.ts#L87)